hono/src/utils/swaggerSchema.ts

```diff
@@ -0,0 +1,300 @@
+import * as fs from 'fs'
+import {
+  OpenAPIRegistry,
+  OpenApiGeneratorV3,
+  extendZodWithOpenApi,
+} from '@asteasolutions/zod-to-openapi'
+import { z } from 'zod'
+
+extendZodWithOpenApi(z)
+
+const registry = new OpenAPIRegistry()
+
+const ItemIdSchema = registry.registerParameter(
```

Comment:
This has the same definition as `ItemIdSchemaDel `. You can keep `ItemIdSchema` and remove the other.